source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,60 @@
+  name: Brandon Casey
+  github: brandonocasey
+tags:
+  - videojs-6
+  - accessibility
+---
+
+Accessibility! The most important feature you never knew about.
+
+In the Video.js organization we try hard to have good accessiblity. Like most other software any change can effect the system in unintended ways. For example `MuteToggle` and `VolumeControl` were recently married into `VolumeMenuButton`. While this change did allow these controls to work in tandem. It also did something unintended. It broke accessibility. In this post we will go over what broke, what the fix was, what accessibility is, and how to test and make sure it works.
+
+> Feel free to skip to the [last section](#The-problem-and-the-solution) if you already know what accessibility is.
+
+### Accessibility? What's that?
+Accessible software has support for users with vision, hearing, or other impairments. Out of the box web applications have some accessibility due to the nature of HTML, but this is only the case if you are using native elements in intended ways. If you cannot use native DOM elements like button, and instead must use a div for buttons. Then you need worry about accessibility in your page.
+
+Supporting hearing impairment is not something that we can do for the users of Video.js. Instead we must indirectly support these users by supporting captions in their videos. In Video.js we have had support for captions/subtitles for some time, internally they are called `TextTracks`.
+
+Supporting vision impairment is harder, but fully in our control. To support this group of users our player must be accessible to "screen readers". A "Screen reader" is an application that reads elements off of the screen to the user (as the name implies). HTML has certain rules that must be followed so that a page can be accessible. We will go over the basics of these rules in the next section.
+
+Here are some popular screen readers that are actually used in the wild:
+
+* [VoiceOver for macOS](https://en.wikipedia.org/wiki/VoiceOver)
+* [JAWS for Windows](https://en.wikipedia.org/wiki/JAWS_(screen_reader))
+
+### How do you make a web application screen reader accessible?
+The most basic way to make a website accessible is to add the `role=""` attribute to an HTML element.
+
+> A list of roles can be found on [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Composite_roles).
+
+Some Examples (assume all of these are plain `div`s):
+* A `MuteToggle` button would have a `role` attribute with a value of `button` like this: `role="button"`
+A `VolumeBar` slider would have a `role` attribute with a value of `slider` like this: `role="slider"`.
+
+If you use the native `<button>` element, you will not have to use the button role. This is the recommended way to make an element a button to the screen reader. In some cases (such as in Video.js) it will not be possible to use the native button element. So you will have to role instead.
```

Comment:
Since `role="button"` with `tabindex` just mimic the `<button>` functionality.
And you still need to bind both space and enter key handlers.